contracts/crowdsale/distribution/RefundableCrowdsale.sol

```diff
@@ -50,10 +62,10 @@ contract RefundableCrowdsale is FinalizableCrowdsale {
     super.finalization();
   }
 
-  // We're overriding the fund forwarding from Crowdsale.
-  // In addition to sending the funds, we want to call
-  // the RefundVault deposit function
-  function forwardFunds() internal {
+  /**
+   * @dev Overrides Crowdsale fund forwarding. In addition to sending the funds, we call the RefundVault deposit function
```

Comment:
I think I wouldn't say `In addition`... we are just overriding to send the funds to the vault instead of sending them to the wallet, right?